hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/RolloutManagementTest.java

```diff
@@ -161,6 +163,105 @@ public void checkRunningRolloutsDoesNotStartNextGroupIfFinishConditionIsNotHit()
     }
 
     @Test
+    @Title("Deleting targets of a rollout")
+    @Description("Verfiying that next group is started when targets of the group have been deleted.")
+    public void checkRunningRolloutsStartsNextGroupIfTargetsDeleted() {
+
+        final int amountTargetsForRollout = 15;
+        final int amountOtherTargets = 0;
+        final int amountGroups = 3;
+        final String successCondition = "100";
+        final String errorCondition = "80";
+        final Rollout createdRollout = createAndStartRollout(amountTargetsForRollout, amountOtherTargets, amountGroups,
+                successCondition, errorCondition);
+
+        finishActionAndDeleteTargetsOfFirstRunningGroup(createdRollout);
+
+        checkStatusOfRolloutGroupsAfterDeletingTargets(createdRollout);
+
+        finishActionAndDeleteTargetsOfSecondRunningGroup(createdRollout);
+
+        deleteThrirdGroup(createdRollout);
+
+        checkStatusOfRolloutAndRolloutGroupsAfterDeletingTargets(createdRollout);
+
+    }
+
+    @Step("Create a rollout by the given parameter and start it.")
+    public Rollout createAndStartRollout(final int amountTargetsForRollout, final int amountOtherTargets,
```

Comment:
All the other tests within this class are doing exactly the same => create rollout and start it. So please call this method from the other tests as well